build/ferretdb/eval-dev.Dockerfile

```diff
@@ -103,8 +103,8 @@ EOF
 FROM ghcr.io/ferretdb/postgres-documentdb-dev:17-ferretdb AS eval-dev
 
 RUN --mount=type=cache,sharing=locked,target=/var/cache/apt <<EOF
-mkdir /tmp/cover /tmp/state
-chown postgres:postgres /tmp/cover /tmp/state
+mkdir /tmp/cover /state
```

Comment:
Why do we need to create `/state` explicitly, if we have `VOLUME` below?

Reply:
Thanks, there is no need, removed this